backend/src/main/java/fk/prof/backend/model/association/BackendDetail.java

```diff
@@ -20,7 +20,7 @@
   private final Set<Recorder.ProcessGroup> associatedProcessGroups;
 
   private long lastReportedTick = 0;
-  //lastReportedTime is null unless backend reports stable load at least once (i.e. currTick reported by backend > 0)
+  //lastReportedTime is null unless backend reports stable load more than once (i.e. currTick reported by backend > 0, since in first load report currTick = 0)
```

Comment:
Yes. The comment incorrectly said at least once. We let first load report go by and update lastreported time from second load report (curr tick > 0)
This is to avoid scenarios where a service goes in crash->restart->crash->restart cycle (because daemon tools will try to keep it up) but still gets marked as available by leader since before crash its able to send first load report (which happens on service bootup)